examples/wav2vec/README.md

```diff
@@ -199,6 +199,48 @@ loss = model(input_values, labels=labels).loss
 loss.backward()
 ```
 
+## Deploying wav2vec 2.0 with torchaudio
+
+`torchaudio` has added wav2vec 2.0 model definition that supports TorchScript, alongside of function to import model instances from `fairseq` or 🤗Transformers. By using TorchScript, you can deploy your wav2vec 2.0 model to ONNX runtime, [C++](https://github.com/pytorch/audio/tree/master/examples/libtorchaudio/speech_recognition), [iOS](https://github.com/pytorch/ios-demo-app/tree/master/SpeechRecognition), and [Android](https://github.com/pytorch/android-demo-app/tree/master/SpeechRecognition).
+
+The following code snipet illustrates the process to import a model and convert it to deployable TorchScript object.
+
+### Import a fine-tuned model from `fairseq`
+
+```python
+import fairseq
+from torchaudio.models.wav2vec2.utils import import_fairseq_model
+
+original, _, _ = fairseq.checkpoint_utils.load_model_ensemble_and_task(
+    ["wav2vec_small_960h.pt"], arg_overrides={'data': "<DIRECTORY_WITH_DICTIONARY>"})
```

Comment:
in newer models, dictionary is actually stored inside the model checkpoint rather than internally. we could probably convert older checkpoints to follow this new format if it will simplify things

Reply:
@alexeib I removed the `arg_overrides`. Did you update the public checkpoints? My concern is that users who just want to try the published model (if not converted to the new format) will encounter an issue when loading models.

Reply:
no, i have not updated the old checkpoints yet. maybe we can allow users to optionally provide data dir instead of making it mandatory to either provide or not?